src/main/java/Main.java

```diff
@@ -0,0 +1,35 @@
+//@Override
+import java.io.IOException;
+
+        import javafx.application.Application;
+        import javafx.fxml.FXMLLoader;
+        import javafx.scene.Scene;
+        import javafx.scene.layout.AnchorPane;
+        import javafx.stage.Stage;
+
+/**
+ * A GUI for Duke using FXML.
+ */
+public class Main extends Application {
+
+    private Duke duke = new Duke();
+
+    public Main() throws IOException, GrapieExceptions {
+    }
```

Comment:
Question: I'm not sure if you need to create a default constructor. My app launches fine without it.